meta-lxatac-software/conf/distro/tacos.conf

```diff
@@ -61,6 +58,8 @@ ICECC_DISABLED ??= "1"
 
 # add poky sanity bbclass
 INHERIT += "poky-sanity"
+ERROR_QA:remove = "patch-status buildpaths"
+WARN_QA:append = "patch-status buildpaths"
```

Comment:
Perhaps limit the change to our layers only? (see https://lists.yoctoproject.org/g/yocto/message/63480)

Reply:
Makes sense. I've changed the respective lines.

Reply:
Hmm, shouldn't they be in the respective layer confs? @ejoerns

Reply:
Sounds good. I've changed the layer configs accordingly.

Reply:
> I don't know which recipe required the `buildpaths` exception, but I guess the CI will tell me soon.

Looks like it was not required after all. I think we are good here.